packages/mui-material-next/src/InputLabel/InputLabel.tsx

```diff
@@ -243,7 +249,7 @@ InputLabel.propTypes /* remove-proptypes */ = {
   /**
    * The variant to use.
    */
-  variant: PropTypes.oneOf(['filled', 'outlined', 'standard']),
-};
+  variant: PropTypes.oneOf(['filled', 'outlined']),
```

Comment:
Probably not in this PR, but we should remember to add the removal of the standard variants on the migration guide.